Examples/Python/python/acts/examples/simulation.py

```diff
@@ -726,6 +654,37 @@ def addGeant4(
     return s
 
 
+def addSimParticleSelection(
```

Comment:
Actually this is quite a lot of code duplication. I would put this in a single function with a string argument, and then maybe add an abstraction interface 

```python
def addParticleSelection(..., kind="simulated" #or "digitized"):
	# impl...

def addSimParticleSelection(*args, **kwargs):
	return addParticleSelection(*args, "simulated", **kwargs)

def addDigiParticleSelection(*args, **kwargs):
	return addParticleSelection(*args, "digitized", **kwargs)
```
or similar.

Would also be fine leaving this to the user totally

Reply:
Yeah good point. This would also remove the cryptic short names `Gen`, `Sim`, `Digi`